src/scripts/globalize.js

```diff
@@ -189,7 +191,11 @@ import { Events } from 'jellyfin-apiclient';
             dictionary = getDictionary(module, fallbackCulture);
         }
         if (!dictionary || !dictionary[key]) {
-            console.error(`Translation key is missing from dictionary: ${key}`);
+            if (dictionary && isEmpty(dictionary)) {
+                console.warn('Translation dictionary is empty.');
+            } else {
+                console.error(`Translation key is missing from dictionary: ${key}`);
+            }
             return key;
         }
         return dictionary[key];
```

Comment:
I tried to do fewer checks :sweat_smile: 
```js
        let dictionary = getDictionary(module, getCurrentLocale());

        if (dictionary) {
            const value = dictionary[key];
            if (value) return value;
        }

        dictionary = getDictionary(module, fallbackCulture);

        if (dictionary) {
            const value = dictionary[key];
            if (value) return value;
        }

        if (dictionary && isEmpty(dictionary)) {
            console.warn('Translation dictionary is empty.');
        } else {
            console.error(`Translation key is missing from dictionary: ${key}`);
        }

        return key;
```

Reply:
Good call that method was in need of a good cleanup!